environments/staging/inventory.ini.j2

```diff
@@ -71,7 +71,7 @@ hqdb1
 # celery0
 
 [hqpillow0]
-hqpillow0-staging.internal-va.commcarehq.org
+# {{ hqpillow0-staging }} hostname=hqpillow0-staging ec2=yes
```

Comment:
We're dropping the "hq" prefix on new aws machines. But also this machine is already outline a couple lines down, so no need to change here.